### DISCUSSION TITLE
CMake Error at runner/CMakeLists.txt:20 (find_package):

### DISCUSSION THREAD

**Discussion by xx-oo** [rustdesk/rustdesk#866] [General]
PS C:\Users\Administrator\rustdesk-flutter_desktop\flutter> flutter run
Multiple devices found:
Windows (desktop) • windows • windows-x64    • Microsoft Windows [版本 10.0.17763.1518]
Chrome (web)      • chrome  • web-javascript • Google Chrome 102.0.5005.115
[1]: Windows (windows)
[2]: Chrome (chrome)
Please choose one (To quit, press "q/Q"): 1
Launching lib\main.dart on Windows in debug mode...
CMake Error at runner/CMakeLists.txt:20 (find_package):
  By not providing "FindCorrosion.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "Corrosion", but CMake did not find one.

  Could not find a package configuration file provided by "Corrosion" with
  any of the following names:

    CorrosionConfig.cmake
    corrosion-config.cmake

  Add the installation prefix of "Corrosion" to CMAKE_PREFIX_PATH or set
  "Corrosion_DIR" to a directory containing one of the above files.  If
  "Corrosion" provides a separate development package or SDK, be sure it has
  been installed.

Building Windows application...
Exception: Unable to generate build files

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/763
